docs/upgrade/canary-upgrade.md

```diff
@@ -2,10 +2,10 @@
 
 ## High level flow
 
-Let us suppose that we have an Istio 1.6 control plane running in our cluster and we would like to upgrade to an Istio 1.7 control plane.
+Let us suppose that we have an Istio 1.6 control plane running in our cluster and we would like to upgrade to an Istio 1.8 control plane.
```

Comment:
```suggestion
Let us suppose that we have an Istio 1.7 control plane running in our cluster and we would like to upgrade to an Istio 1.8 control plane.
```
And similarly for the rest of this doc.

Reply:
There are other such cases in both canary-upgrade.md and in-place-upgrade.md

Reply:
👍